contrib/completion/zsh/_docker

```diff
@@ -641,6 +641,7 @@ __docker_subcommand() {
             _arguments \
                 $opts_help \
                 "($help -c --change)*"{-c,--change=-}"[Apply Dockerfile instruction to the created image]:Dockerfile:_files" \
+                "($help -m --message)"{-m,--message=}"[Set commit message for imported image]:message: " \
```

Comment:
It's just for consistency. We should in fact remove the `-` from everywhere as docker accepts arguments without `=`.